README.md

```diff
@@ -1,284 +1,126 @@
-# `@cloudflare/next-on-pages`
+# ⚡▲ next-on-pages ▲⚡
 
-Reference:
+next-on-pages is a CLI tool that you can use to build and develop [Next.js](https://nextjs.org/) applications so that they can run on the [Cloudflare Pages](https://pages.cloudflare.com/) platform (and integrate with the various Cloudflare offerings such as KV stores, and Durable Objects).
 
-- [Blog](https://blog.cloudflare.com/next-on-pages)
-- [Documentation](https://developers.cloudflare.com/pages/framework-guides/deploy-a-nextjs-site/)
+This tool is a best-effort library implemented by the Cloudflare team and the community since Vercel doesn't and likely won't support Next.js running on Cloudflare officially. As such most but not all Next.js features are yet supported, see the [Supported Versions and Features document](./docs/supported.md) for more details.
 
 ## Quick Start
 
-1. `npx create-next-app@latest my-app`
+This section describes how to bundle and deploy a (new or existing) Next.js application and using next-on-pages.
 
-   Note that if you elect to use eslint, there are a couple of places you need to add return types to make the default template pass the pre-build checks.
+> Note: this readme uses `npm`, but nothing relies on it, you can instead use `yarn` or `pnpm` if you want.
 
-1. `cd` into the new directory (e.g. `cd my-app`)
+### 1. Create Next App
 
-1. `npm install -D @cloudflare/next-on-pages vercel`
+To start creating a Next.js application using next-on-pages, start by creating a standard Next.js application using the create-next-app command (skip this step if you already have an existing Next.js application that you want to port to Cloudflare using next-on-pages):
 
-1. Configure the project to use the Edge Runtime:
+```sh
+npx create-next-app@13.2.4 my-next-app
+```
 
-   Replace `pages/api/hello.ts` with the following:
```

Comment:
I don't think we can really tell the developer what to change since the create-next-app offers the option to use `src`, `app` or neither, so I changes the message to be less specific (I think that adding details for all possible choices would be way too verbose)